components/Core/Divider/index.tsx

```diff
@@ -0,0 +1,10 @@
+import { FC, HTMLAttributes } from 'react'
+import styles from './divider.module.css'
+
+const Divider: FC<HTMLAttributes<HTMLElement>> = () => {
```

Comment:
a good way to gain capability to extend these simple components is to spread in props passed in, example:

```
const Divider = ({myAwesomeProp, ...restProps}) => (<hr {...restProps}/>)

// Now if I use this somewhere I can do something like

<Divider aria-label="this is a divider" />

// and it really does pass the aria label in to the underlying component
```

Reply:
Yes, but having the props defined this way shows you autocomplete when you are passing props to it.